internal/service/autoscaling/policy_test.go

```diff
@@ -590,8 +644,78 @@ resource "aws_autoscaling_policy" "test" {
 `, name))
 }
 
-func testAccPolicyConfig_predictiveScalingRemoved(name string) string {
-	return acctest.ConfigCompose(testAccPolicyConfig_base(name), fmt.Sprintf(`
+func testAccPolicyConfigPredictiveScalingCustom(name string) string {
+	return acctest.ConfigCompose(testAccPolicyConfigBase(name), fmt.Sprintf(`
+resource "aws_autoscaling_policy" "test" {
+  name                   = "%[1]s-policy_predictive"
+  policy_type            = "PredictiveScaling"
+  autoscaling_group_name = aws_autoscaling_group.test.name
+  predictive_scaling_configuration {
+    metric_specification {
+      target_value = 32
+      customized_scaling_metric_specification {
```

Comment:
Small styling nit: can we order these "block" arguments for easier review

Reply:
Could you be more specific what's expected order ?